.eslintrc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
     root: true,
-    extends: '@oat-sa/eslint-config-tao'
+    extends: '@oat-sa/eslint-config-tao/node'
```

Comment:
Why did you choose the node config? Presumably all this code runs in the browser. (I checked our configs for [node](https://github.com/oat-sa/eslint-config-tao/blob/master/node.js) and [svelte/ESM](https://github.com/oat-sa/eslint-config-tao/blob/master/svelte.js) - and there is no difference in rules for *.js files. Well, the latter has `env.browser = true`, and the parser is a different one. Is this part of your reasoning?)

Reply:
Yes, good catch, thank you! Indeed, the code will run in the browser, but it will also pass through a bundling step, and I needed to have support for ESM and Jest. The Svelte variant looked too specific for me, with the use of a plugin that is not needed for this repo (eslint-svelte3). Anyway, it looks to work without, so I'll change the config.